articles/criticalhit.md

```diff
@@ -55,3 +58,21 @@ Gen 1-5:
 | +2    | 1/4    |      25.0% |
 | +3    | 1/3    |      33.3% |
 | +4    | 1/2    |      50.0% |
+
+Gen 2 only:
+
+The attacking stat stage and defensive stat stage were considered as a collective unit, and either both treated as 0 or both left as their full value, depending on whether their sum was less than 0. 
+
+High-critical-rate moves were given a starting ratio of +2 rather than +1.
+
+Gen 1:
+
+The critical modifier of 2x was applied onto the level, as opposed to the final damage. As such, at level 100 it effectively became a multiplier of 82/42x, or about 1.95x. 
+
+Critical hits always caused all stat stages on both sides to be treated as 0, regardless of whether that was slated to result in more or less damage.
+
+Critical rate was calculated by default as the attacker's base speed divided by 512.
```

Comment:
Technically, if you're going to do this out of 256, then it's effectively just `floor(base Speed / 2)` (but actually `floor(floor(floor(base Speed / 2) * 2) /2)`). It seems like you're trying to do it in percentages, which is commonly listed as (base Speed / 512) * 100, but in reality that's off by fractions of a percent because the original base Speed is halved in the first step of crit calculation. For Focus Energy it's `floor(floor(floor(base Speed / 2) / 2) /2)`.

That's all for regular moves; high-crit moves multiply by 4 instead of dividing by 2 in the last step above, capping at 255.

Reply:
It's not – if we do it at all, it should be in footnotes or something.